src/readit/details/comments/index.js

```diff
@@ -289,8 +288,8 @@ function Details(
                             <hr/>
 
                             <h2>Ratings & Reviews</h2>
-
                             <div className="row">
+                                {currentUser &&
```

Comment:
only going to show reviews on this page if they are logged in if that's cool

Reply:
is there a reason for only showing reviews when logged in? I was thinking of putting it in public profiles but wouldn't make sense to if we're hiding them here

Reply:
I think just as part of the not logged in/logged in information hiding, but I can put it back for sure

Reply:
actually this is just blocking posting a review